src/components/nve-textarea/nve-textarea.component.ts

```diff
@@ -117,11 +116,13 @@ export default class NveTextarea extends LitElement implements INveComponent{
   /** Hoved input felt i nve-textarea komponentet. Brukes til constraint validering */
   @query('.textarea__control') input!: HTMLTextAreaElement;
 
+  //Brukes for "for"-attributten på nve-label.
+  private textAreaid: string = '';
```

Comment:
Lurer bare på om det kan være et scenario hvor folk vil bestemme selv hva slags verdi id burde ha, men tipper vi kan ha det slik for nå, og evt tilpasse hvis det trengs